src/components/TransactionItem/TransactionItemCell/index.tsx

```diff
@@ -90,10 +90,26 @@ const WithdrawPopoverItem = ({
   </WithdrawItemPanel>
 )
 
-const WithdrawPopoverInfo = ({ cell }: { cell: State.Cell }) => {
+export const WithdrawPopoverInfo = ({ cell, cellV2 }: { cell: State.Cell; cellV2?: State.Transfer }) => {
   const isMobile = useIsMobile()
   const { app } = useAppState()
   let width = 'short'
+
+  // Compatible with V2 interface
+  if (cellV2) {
+    /* eslint-disable */
```

Comment:
Please specify the rule explicitly so we can fix it easily later

Reply:
The point here is to add a clear rule of eslint to disable, e.g. `/* eslint-disable rule-1 */`, so we know what should be refactored here.

Reply:
The error message is from ESLint's 'no-param-reassign' rule, which prohibits reassigning values to function parameters.

The code has been modified to prevent the error from occurring
```ts
// Compatible with V2 interface
  const updatedCell = {
    ...cell,
    ...(cellV2 && {
      cellType: cellV2.transferType,
      capacity: cellV2.capacity.toString(),
      interest: cellV2.interest ? cellV2.interest.toString() : '',
      compensationStartedBlockNumber: cellV2.compensationStartedBlockNumber || 0,
      compensationEndedBlockNumber: cellV2.compensationEndedBlockNumber || 0,
      compensationStartedTimestamp: cellV2.compensationStartedTimestamp || 0,
      compensationEndedTimestamp: cellV2.compensationEndedTimestamp || 0,
      lockedUntilBlockNumber: cellV2.lockedUntilBlockNumber || 0,
      lockedUntilBlockTimestamp: cellV2.lockedUntilBlockTimestamp || 0,
    }),
  }
```
In the subsequent code, use the 'updatedCell' object.